test/bdd/fixtures/discovery-config/sidetree-fabric/config-data/peer0-org1.trustbloc.local.json

```diff
@@ -4,9 +4,16 @@
   "policy": {
     "cache": {"max_age": 123456789}
   },
-  "endpoints": [
-    "https://peer0-org1.trustbloc.local/document",
-    "https://peer1-org1.trustbloc.local/document"
+  "members": [
```

Comment:
why a consortium file?

Reply:
we don't have load balancer locally we use peer container name as domain

Reply:
done